packages/core/src/erc721.ts

```diff
@@ -126,6 +132,24 @@ function addMintable(c: ContractBuilder, access: Access, incremental = false, ur
   }
 }
 
+function addVotes(c: ContractBuilder, name: string) {
+  c.addParent(
+    {
+      name: 'ERC721Votes',
+      path: '@openzeppelin/contracts/token/ERC721/extensions/draft-ERC721Votes.sol',
+    });
+  c.addParent(
+    {
+      name: 'EIP712',
+      path: '@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol',
+    },
+    [name, { lit: '"1"' }],
```

Comment:
Strings are printed with quotes (see [printValue](https://github.com/OpenZeppelin/contracts-wizard/blob/de501648fcc7ea313de179ab5550fff37815eb24/packages/core/src/print.ts#L139)).

```suggestion
    [name, "1"],
```